gradle-sls-packaging/src/main/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -85,8 +86,14 @@ public void apply(Project project) {
                     task.setAppendix("manifest-classpath");
 
                     task.doFirst(t -> {
-                        String classPath = project.getConfigurations().getByName("runtimeClasspath")
-                                .getFiles()
+
+                        FileCollection runtimeClasspath = project.getConfigurations().getByName("runtimeClasspath");
+
+                        TaskProvider<Jar> jarTask = project.getTasks().withType(Jar.class)
+                                .named(JavaPlugin.JAR_TASK_NAME);
```

Comment:
could we inline this into the below? also can do `getByName` rather than `named` if you're just gonna `.get()` it right afterwards